assets/styles/styles.css

```diff
@@ -0,0 +1,347 @@
+body {
+  background-color: #f4f4f7;
+
+  /* background-color: seagreen;  */
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+.sec {
+  /* width: 500px;
+   min-width: 200px; */
+  min-height: 20vh;
+}
+
+.sec1 {
+  background-color: seagreen;
+}
+
+.sec2 {
+  background-color: sienna;
+}
+
+.sec3 {
+  background-color: slateblue;
+}
+
+.sec4 {
+  background-color: rgb(122, 121, 156);
+}
+
+.sec5 {
+  background-color: rgba(235, 11, 255, 0.945);
+}
+
+.twot {
+  /* margin-top: 100px !important; */
+}
+
+img {
+  /* width: 100%;  */
+}
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */